zipline/pipeline/graph.py

```diff
@@ -70,7 +70,9 @@ def offset(self):
 
             Max number of extra rows needed by any term depending on `input`
             minus
-            Number of extra rows needed by `term`.
+            Number of rows needed by `input` for `term`
```

Comment:
Can you explain a bit more about why this is necessary?  I don't have much intuition for why this is correct just from reading it.